lib/puppet/type/file.rb

```diff
@@ -51,7 +51,7 @@ def self.title_patterns
 
     validate do |value|
       unless Puppet::Util.absolute_path?(value)
-        fail Puppet::Error, "File paths must be fully qualified, not '#{value}'"
+        fail Puppet::Error, "File paths must be fully qualified"
```

Comment:
This line of code doesn't work, fail does not expand value. Also, the value
string is already reported in the message header.


On Thu, Jul 21, 2016 at 10:54 AM, Kylo Ginsberg <notifications@github.com>
wrote:

> In lib/puppet/type/file.rb
> <https://github.com/puppetlabs/puppet/pull/5126#discussion_r71753984>:
>
> > @@ -51,7 +51,7 @@ def self.title_patterns
> >
> >      validate do |value|
> >        unless Puppet::Util.absolute_path?(value)
> > -        fail Puppet::Error, "File paths must be fully qualified, not '#{value}'"
> > +        fail Puppet::Error, "File paths must be fully qualified"
>
> Why drop the path from the message?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/puppetlabs/puppet/pull/5126/files/883ba1c7aa3a0935d40f984175dd76b0f1352374#r71753984>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMVyKiWGe_YSAaiqUCeQUjkDpikV7qW2ks5qX7JAgaJpZM4JQCqN>
> .
>



-- 
*Andy Henroid*
Senior Software Engineer
andy.henroid@puppet.com